log/slow/parser.go

```diff
@@ -358,7 +358,11 @@ func (p *SlowLogParser) sendEvent(inHeader bool, inQuery bool) {
 		// Started parsing in header after Query_time.  Throw away event.
 		return
 	}
-
+	// If the query was logged because it took more than RateMaxQueryTime
+	// the rate should be ignored to prevent a wrong sampling
+	if p.opt.RateMaxQueryTime > 0 && p.event.TimeMetrics["Query_time"] > p.opt.RateMaxQueryTime {
+		p.event.RateLimit = 0
```

Comment:
This seems weird. I thought we wanted to skip events, not change the rate limit?